From 75c7c219ccf613f678c9fe84868d7a76691b9dc9 Mon Sep 17 00:00:00 2001
From: Huang Chenyi <b44402@freescale.com>
Date: Mon, 6 May 2013 14:32:33 +0800
Subject: [PATCH 194/602] ENGR00261418 add a flag "encryptable=footer" in
 fstab.freescale

This flag is what tells android that the filesystem can be encrypted.

Signed-off-by: Huang Chenyi <b44402@freescale.com>
---
 imx6/etc/fstab.freescale | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/etc/fstab.freescale b/imx6/etc/fstab.freescale
index 685b4ea..004bc46 100644
--- a/imx6/etc/fstab.freescale
+++ b/imx6/etc/fstab.freescale
@@ -5,7 +5,7 @@
 
 
 /dev/block/mmcblk0p5    /system  ext4    ro                                                                               wait
-/dev/block/mmcblk0p4    /data    ext4    nosuid,nodev,nodiratime,noatime,nomblk_io_submit,noauto_da_alloc,errors=panic    wait
+/dev/block/mmcblk0p4    /data    ext4    nosuid,nodev,nodiratime,noatime,nomblk_io_submit,noauto_da_alloc,errors=panic    wait,encryptable=footer
 /dev/block/mmcblk0p6    /cache   ext4    nosuid,nodev,nomblk_io_submit				                          wait
 /dev/block/mmcblk0p7    /device  ext4    ro,nosuid,nodev				                                  wait
 
-- 
1.8.0

